mlos_core/mlos_core/optimizers/bayesian_optimizers/__init__.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Basic initializer module for the mlos_core Bayesian optimizers.
+"""
+
+from mlos_core.optimizers.bayesian_optimizers.bayesian_optimizer import BaseBayesianOptimizer
+from mlos_core.optimizers.bayesian_optimizers.emukit_optimizer import EmukitOptimizer
+from mlos_core.optimizers.bayesian_optimizers.skopt_optimizer import SkoptOptimizer
+
+
+__all__ = [
+    'BaseBayesianOptimizer',
+    'EmukitOptimizer',
+    'SkoptOptimizer',
```

Comment:
- [ ] May want to add test cases that all subclasses of `BaseOptimizer` class are available in the factory methods, enum classes, etc.
  (this is relevant in the config test automation for `mlos_bench` as well - #340)